mypy/main.py

```diff
@@ -399,7 +411,7 @@ def _print_message(self, message: str, file: SupportsWrite[str] | None = None) -
         if message:
             if file is None:
                 file = self.stderr
-            file.write(message)
+            print(message, file=file)
```

Comment:
I wonder if I'll have to add end='' to this...
```suggestion
            print(message, file=file, end='')
```

Reply:
Ah, yes.